views/statusboard.ejs

```diff
@@ -1,92 +1,101 @@
-        <th class="sort" data-sort="license">license</th>
-        <th class="sort min" data-sort="fork">fork</th>
-        <th class="sort min" data-sort="fork-count">fork count</th>
-        <th class="sort" data-sort="test">test</th>
-        <th class="sort" data-sort="size">size</th>
-        <th class="sort min" data-sort="stars">stars</th>
-        <th class="sort" data-sort="last-commit" style="min-width: 200px;">last commit date</th>
-        <th class="sort" data-sort="last-release" style="min-width: 200px;">last release date</th>
-        <th class="sort" data-sort="unreleased-commit">unreleased commit</th>
-        <th class="sort min" data-sort="pr">pr</th>
-        <th class="sort min" data-sort="issues">issues</th>
-        <th class="sort min" data-sort="nyc">cov lib</th>
-        <th class="sort min" data-sort="dep">dep</th>
-        <th class="sort min" data-sort="devDep">devDep</th>
-        <th class="sort min" data-sort="branch">branch</th>
-      </tr>
-    </thead>
-    <tbody class="list">
-      <% for (const project of projects) { %>
-      <tr>
-        <td class="name left" style="min-width: 170px">
-          <a href="<%= project.url %>" target="_blank">
-            <%= project.name %>
-          </a>
-        </td>
-        <td class="pkg-name left" style="min-width: 200px"><%= project.package_name %></td>
-        <td class="version"><%= project.version %></td>
-        <td class="private"><%= project.private ? "✔️" : "❌" %></td>
-        <td class="node"><%= project.nodejs_version %></td>
-        <td class="esm"><%= project.is_module ? "✔️" : "❌" %></td>
-        <td class="license" style="min-width: 100px"><%= project.license %></td>
-        <td class="fork"><%= project.fork ? "✔️" : "❌" %></td>
-        <td class="fork-count"><%= project.fork_count %></td>
-        <td class="test"><%= project.test_framework %></td>
-        <% if (project.private) { %>
-        <td class="size"></td>
-        <% } else { %>
-        <td class="size">
-          <img src="https://img.shields.io/github/languages/code-size/<%= orgName %>/<%= project.name %>"></img>
-        </td>
-        <% } %>
-        <td class="stars"><%= project.stars %></td>
-        <td class="last-commit" data-timestamp="<%= new Date(project.last_commit.date).getTime() %>"><%= project.last_commit.date %></td>
-        <td class="last-release" data-timestamp="<%= new Date(project.last_release).getTime() %>"><%= project.last_release %></td>
-        <td class="unreleased-commit" ><%= project.unreleased_commit_count %></td>
-        <td class="pr"><%= project.pull_request.length %></td>
-        <td class="issues"><%= project.issues.length %></td>
-        <td class="nyc"><%= project.coverage_lib %></td>
-        <td class="dep"><%= project.dependencies_count %></td>
-        <td class="devDep"><%= project.dev_dependencies_count %></td>
-        <td class="branch"><%= project.default_branch %></td>
-      </tr>
+<table>
```

Comment:
```suggestion
  <table>
```
(you need to add 2 spaces of indentation for the whole content inside the `div.table-container`